pkg/commands/options/publish.go

```diff
@@ -73,6 +78,9 @@ func AddPublishArg(cmd *cobra.Command, po *PublishOptions) {
 	if dockerRepo, exists := os.LookupEnv("KO_DOCKER_REPO"); exists {
 		po.DockerRepo = dockerRepo
 	}
+	if dockerRepoHost, exists := os.LookupEnv("KO_DOCKER_REPO_ADDRESS"); exists {
```

Comment:
Can the same thing be achieved with `HTTP_PROXY`?

Reply:
I don't **think** so, since HTTP_PROXY should be a completely different machinery. I can try it out tho. Do you have something specific in mind, or just setting HTTP_PROXY to the "actual" location of the registry (the value I'd set `KO_DOCKER_REPO_ADDRESS` to in my world)?

Reply:
Tried it, doesn't work as expected.